blkn/src/constants.js

```diff
@@ -30,3 +30,9 @@ export const fontColors = {
   error: colors.red,
   input: colors.casper,
 };
+
+export const fontSizes = {
```

Comment:
😄 at least sort it by size or something

Reply:
This was copied from an older version of the Typography component. :-)

I would rather see the Typography component used in the Button.

Reply:
Well first i was instructed to not import other components, so i could create the copy of typography component just for this one purpose but I see no reason tbh

Reply:
It must be misunderstanding, I didn't instruct you don't import other components.I meant don't reuse IconWrapper as the universal component. Because in my opinion, IconWrapper is just helper component for Button. If you try It makes so universal it will be quickly overwhelmed.

Reply:
ok . I will use ```Typography``` then :)